src/components/OverviewApp/Notary.tsx

```diff
@@ -155,14 +155,23 @@ const Notary = (props: { notaryProps: NotaryProps }) => {
     }
   };
 
-
   const checkAlreadyProposed = async (issueNumber: number) => {
-    const { data } = await context.github.githubOcto.issues.listComments({
+    const issue = await context.github.githubOcto.issues.get({
       owner: config.onboardingLargeOwner,
       repo: config.onboardingLargeClientRepo,
       issue_number: issueNumber
     });
 
+    const pagination = Math.floor(issue.data.comments / 100) + 1
+
+    const { data } = await context.github.githubOcto.issues.listComments({
```

Comment:
to do this don't need to use pagination

this is an example to fetch all the issues
let issuez = await octokit.paginate(octokit.issues.listForRepo, {
      owner,
      repo,
      state: "open",
    });
    
    there are other examples around the code
    here docs: https://octokit.github.io/rest.js/v18#pagination

Reply:
https://octokit.github.io/rest.js/v18#issues-list-comments

Reply:
added paginate function